test/cgroups.bats

```diff
@@ -28,7 +28,7 @@ function teardown() {
 }
 
 @test "conmon custom cgroup" {
-	CONTAINER_CGROUP_MANAGER="systemd" CONTAINER_CONMON_CGROUP="customcrioconmon.slice" start_crio
+	CONTAINER_CGROUP_MANAGER="systemd" CONTAINER_DROP_INFRA_CTR=false CONTAINER_MANAGE_NS_LIFECYCLE=false CONTAINER_CONMON_CGROUP="customcrioconmon.slice" start_crio
```

Comment:
since `false` is the default for these two, why set them?

Reply:
the reason is to future proof the test. These tests require we don't drop infra, regardless of what the default is. Setting them here decreases the difficulty of changing the default

also, manage_ns_lifecycle is true by default

Reply:
would you rather I don't set it @kolyshkin